fixtures/test-fixtures.sh

```diff
@@ -25,6 +25,17 @@ run_fixture() {
 }
 
 main() {
+  [ $# -ne 0 ] && for fixture in "$@"; do
+    (run_fixture "$fixture")
+    exit_status=$?
+    if [ "$exit_status" -eq 0 ]; then
+      echo -e "[${GREEN}ok${NC}] $fixture"
+    else
+      echo -e "[${RED}fail${NC}] $fixture"
+      exit "$exit_status"
+    fi
```

Comment:
I like the idea. I've done something similar on my dev box.

But I am not a fan of the code duplication. How about using a function, e.g. `output_result "$fixture" $?` for lines 30-36.

Then you can also use that function below (line 43).